Utilities/HttpServer.cs

```diff
@@ -246,6 +333,7 @@ public class HttpServer {
       JSON += "]";
 
       if (n is SensorNode) {
+        JSON += ", \"id\": \"" + ((SensorNode)n).Sensor.Identifier.ToString() + "\"";

```

Comment:
I think we should follow the other output here and use "Id".

Reply:
Agreed, fixed.